server/src/Types.hs

```diff
@@ -213,6 +237,27 @@ instance Docs.ToSample HashedScript where
       )
     ]
 
+instance Docs.ToSample FinalizeRequest where
```

Comment:
Can you update the server readme with this please? You can use `markdown apiDocs` and copy-paste the new endpoint docs